api/src/pcapi/core/finance/api.py

```diff
@@ -1378,12 +1381,22 @@ def get_individual_data(query: BaseQuery) -> BaseQuery:
                 offerers_models.Offerer.name.label("offerer_name"),
                 offerers_models.Offerer.siren.label("offerer_siren"),
                 models.Deposit.type.label("deposit_type"),
-                sqla_func.sum(models.Pricing.amount).label("pricing_amount"),
+                sa.case((offerers_models.Offerer.is_caledonian == True, "NC"), else_=None).label("is_caledonian"),
+                sqla_func.sum(models.Pricing.xpf_amount if only_caledonian else models.Pricing.amount).label(
+                    "pricing_amount"
+                ),
             )
         )
 
+        if only_caledonian:
+            individual_data_query = individual_data_query.filter(
+                offerers_models.Offerer.is_caledonian == only_caledonian
+            )
```

Comment:
A priori tu peux simplifier : 
```suggestion
            individual_data_query = individual_data_query.filter(offerers_models.Offerer.is_caledonian)
```